switchover/dbsync/dbid.go

```diff
@@ -0,0 +1,15 @@
+package dbsync
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+)
+
+func newDBID() string {
```

Comment:
Would you mind walking me through how this is leveraged?  From `switchover/dbsync/sync.go` it looks like it's being used to create both `oldDbID` and `newDbID`, so I'm confused how it's leveraged in the context of config verification.

Reply:
Hmm. Yeah, I suppose that is a little confusing. The "new" in the context of this function is just referring to creation, like `http.NewServeMux` creates a new `http.ServeMux`.

Inside `sync.go` oldDB things refer to the `--db-url` database, the one being switched away from. newDB things refer to the `--db-url-next`, or the one we are switching _to_.

Before we call it an official/hardened feature, it might be worth coming up with a better naming scheme for tracking the two databases, rather than old and new.

As far as the purpose of this function, each time the shell starts it generates a random ID for each DB, and publishes that ID to a channel on only that DB. It serves as a functional test that what the shell thinks is the next/new DB is the same one as what each GoAlert instance thinks is the next/new DB. Same for the old/prev DB.

Does that help?

Reply:
Yes - thank you!